src/twisted/internet/test/test_process.py

```diff
@@ -862,6 +862,8 @@ class PTYProcessTestsBuilder(ProcessTestsBuilderBase):
     if platform.isWindows():
         skip = "PTYs are not supported on Windows."
     elif platform.isMacOSX():
+        skip = "PTYs are flaky from a Darwin bug. See #8840."
```

Comment:
I had something that shouldn't be related to close crash, though... maybe i need to stress test them individually. Does this happen more on faster PCs than slower? I tried reproducing and it wouldn't...